src/store.js

```diff
@@ -0,0 +1,15 @@
+import { configureStore, getDefaultMiddleware } from '@reduxjs/toolkit';
+import logger from 'redux-logger';
+import thunk from 'redux-thunk';
+import reducer from './components/DataSlice';
+
+const rootReducer = {
+  rootReducer: reducer,
+};
+
+const store = configureStore({
+  reducer: rootReducer,
```

Comment:
[OPTIONAL] Consider creating a folder for all redux logic ie to store this store file and all other slices to add a bit more structure to your application

Reply:
Done